apis/nucleus/src/components/listbox/useConfirmUnfocus.jsx

```diff
@@ -15,6 +15,8 @@ export default function useConfirmUnfocus(ref, selections) {
   };
 
   useEffect(() => {
+    if (!shouldConfirmOnBlur) return () => {};
```

Comment:
there should be no need to return an empty function here
I would go with just `return;` (or `return undefined;` if a return value are needed by lint rules)

Reply:
ok I'll try to remove it